astroid/brain/__init__.py

```diff
@@ -0,0 +1,3 @@
+__all__ = ["register_module_extender"]
+
+from astroid.brain.register_module_extender import register_module_extender
```

Comment:
This isn't necessary. Nobody should use `from astroid.brain import *` in the first place. There isn't a need for it.